client/src/components/generic/Event/EventDetailed/EventDetailed.tsx

```diff
@@ -0,0 +1,139 @@
+import Image from "next/image"
+/**
+ * Props for event card
+ */
+export interface IEventDetailed {
+  /**
+   * The date of the event as a string.
+   * @example "THU 18/7 • 6pm"
+   */
+  date?: string
+
+  /**
+   * Link of google form to embed into the site
+   */
+  googleFormLink?: string
+
+  /**
+   * If the event is over, to avoid the user signing up etc
+   */
+  isPastEvent?: boolean
+
+  /**
+   * When the signups open (includes both _time_ and _date_),
+   * this prop will have a side-effect on if the {@link googleFormLink}
+   * is displayed
+   */
+  signUpOpenDate?: Date
+
+  /**
+   * The title of the event as a string
+   * @example "I am sigma"
+   */
+  title: string
+
+  /**
+   * The location of the event as a string
+   * @example "Basement"
+   */
+  location?: string
+
+  /**
+   * The content of the event as a react node
+   */
+  content: React.ReactNode
+
+  /**
+   * The callback for when the user attempts to return to any previous
+   * view (e.g the list of events)
+   */
+  onBack: () => void
+
+  /**
+   * The image cover on the top of the card
+   */
+  image: string
+}
+
+/**
+ * The view for users to sign up and see full information about the event.
+ *
+ * **No data fetching should be performed in this component**
+ */
+const EventDetailed = ({
+  date,
+  title,
+  location,
+  content,
+  onBack,
+  image,
+  signUpOpenDate,
+  googleFormLink,
+  isPastEvent
+}: IEventDetailed) => {
+  const Divider = () => {
+    return <div className="bg-gray-3 mb-4 mt-4 h-[1px] w-full"></div>
+  }
+  return (
+    <>
+      <h5
+        onClick={onBack}
+        className="text-dark-blue-100 cursor-pointer self-start text-lg font-bold uppercase"
+      >
+        Back
+      </h5>
+      <div className="flex w-full flex-col items-center justify-center gap-4">
+        <div className="flex h-fit w-full justify-center border bg-white">
+          <Image
+            src={image}
+            alt="Event cover image"
+            width={500}
+            height={300}
+            className="block h-auto max-h-[400px] w-auto  py-4"
+          />
+        </div>
```

Comment:
<img width="1053" alt="image" src="https://github.com/user-attachments/assets/cdf88525-2803-4536-bcf2-5b97ff0b9428">

Is it intentional that the border doesn't wrap the image and overlaps in a rectangle? Thought just as an improvement, could just have the image centre instead.

Reply:
the image should ideally fill the container so yes its not intentional, will need to be resolved

Reply:
done